magenta/models/sketch_rnn/sketch_rnn_train.py

```diff
@@ -429,8 +429,8 @@ def trainer(model_params):
 
   tf.logging.info('sketch-rnn')
   tf.logging.info('Hyperparams:')
-  for key, val in six.iteritems(list(model_params.values())):
-    tf.logging.info('%s = %s', key, str(val))
+  # for key, val in six.iteritems(list(model_params.values())):
```

Comment:
can you either fix these lines or remove them?

Reply:
Sure thing! I'll remove and recommit.

Reply:
Done! :+1: